server/dash/dashboards/ncSumComp.py

```diff
@@ -0,0 +1,407 @@
+import datetime
+import dash_daq as daq
+import numpy as np
+import pandas as pd
+import plotly.express as px
+import plotly.graph_objects as go
+import requests as re
+
+from config import API_HOST
+from design import DISCRETE_COLORS, LABELS, apply_figure_style
+from dash import dcc, html, callback
+from dash.dependencies import Input, Output
+from dash.exceptions import PreventUpdate
+
+# Setting 1 week worth of data
```

Comment:
Please add periods at the end of all comments.

Reply:
Thanks for the catch - resolved.